Wikimate.php

```diff
@@ -196,8 +205,9 @@ public function query( $array ) {
 
 	/**
 	 * Performs a parse query to the wiki API.
-	 * @param array $array array of details to be passed in the query.
-	 * @return array unserialized php output from the wiki API.
+	 *
+	 * @param   array  $array  Array of details to be passed in the query.
```

Comment:
All other param lines have no dot at the end - trim them in these two lines too?